apps/ensindexer/src/lib/seaport/seaport-helpers.ts

```diff
@@ -0,0 +1,36 @@
+import { Context } from "ponder:registry";
+import schema from "ponder:schema";
+import { Address, Hex } from "viem";
+
+/**
+ * Lookup the domain id for a tokenId
+ * @param context - ponder context
+ * @param contractAddress - contract address of the NFT
+ * @param tokenId - tokenId of the NFT
+ */
+export async function lookupDomainId(
+  context: Context,
+  contractAddress: Address,
+  tokenId: string,
+): Promise<Hex | null> {
+  const tokenIdHex = `0x${BigInt(tokenId).toString(16).padStart(64, "0")}` as Hex;
+
+  // OLD ENS Registry: tokenId is labelhash, need to find domain by labelhash
+  if (contractAddress === "0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85") {
```

Comment:
It'd be great to replace this inline contract address value with a reference to that value from `@ensnode/datasources`.

For example, you can get this address in a following way:
```ts
import { DatasourceNames, getDatasource } from "@ensnode/datasources"
import config from "@/config";

/** ENSRoot Base Registrar Contract Address */
const baseRegistrarContractAddress = getDatasource(config.namespace, DatasourceNames.ENSRoot)
  .contracts["BaseRegistrar"].address;
```

and then use the `baseRegistrarContractAddress` reference 🚀